torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1049,6 +1049,12 @@ that adds a prefix to each key inserted to the store.
               py::arg("opts") = ::c10d::BarrierOptions(),
               py::call_guard<py::gil_scoped_release>());
 
+  // base ProcessGroup::Options binding
+  auto processGroupOptions =
+      intrusive_ptr_class_<::c10d::ProcessGroup::Options>(
+          module, "ProcessGroupOptions")
+          .def_readwrite("backend", &::c10d::ProcessGroup::Options::backend);
```

Comment:
It looks like the `10d::ProcessGroup::Options` also has a `timeout`, should we add that binding here as well?

Reply:
I thought I added it already lol, weird it doesn't trigger any failures